bin/all_sky_search/pycbc_bin_trigger_rates_dq

```diff
@@ -37,16 +37,17 @@ parser.add_argument("--prune-window", type=float, default=0.1,
                          "which is loudest in each split, default 0.1s")
 
 pystat.insert_statistic_option_group(parser,
-    default_ranking_statistic='single_ranking_only')
+        default_ranking_statistic='single_ranking_only')
 args = parser.parse_args()
 pycbc.init_logging(args.verbose)
 
 with h5.File(args.bank_file, 'r') as bank:
     if args.background_bins:
         logging.info('Sorting bank into bins...')
         data = {'mass1': bank['mass1'][:], 'mass2': bank['mass2'][:],
-                'spin1z': bank['spin1z'][:], 'spin2z': bank['spin2z'][:],
-                'f_lower': bank['f_lower']}
+                'spin1z': bank['spin1z'][:], 'spin2z': bank['spin2z'][:]}
+        # hardcoding low frequency cutoff to 15 Hz for uniformity
```

Comment:
Sorry, I changed my mind.  Better to make this a command line option with a default of 15. as otherwise it will be messy to retune if wanted.

Reply:
You can move the comment as-is to the argparse section ;)

Reply:
Ok I have made it a config option